src/ocm/components/clusterDetail/ClusterDetail.tsx

```diff
@@ -49,7 +50,16 @@ const ClusterDetail: React.FC<ClusterDetailProps> = ({ cluster }) => {
   const { resetClusterDialog, cancelInstallationDialog } = useModalDialogsContext();
   const clusterVarieties = useClusterStatusVarieties(cluster);
   const { credentials, credentialsError } = clusterVarieties;
-  const canAddHosts = !isSNO(cluster) && cluster.status === 'installed' && !ocmClient;
+  const featureSupportLevelContext = useFeatureSupportLevel();
+  const canAddHosts =
```

Comment:
We have very similar logic in `isSNOExpansionAllowed` so we should be having it in only one place and reuse it.

Please be careful that the function `canAddHost` in the same file is being used from OCM so it should not be changed.

Reply:
But the type of Clusters are differents. isSNOExpansionAllowed in src/components/AddHosts/utils.ts are using OcmClusterType(day2 cluster) and in this case we use Cluster (day1 cluster).